SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialRPCService.h

```diff
@@ -79,23 +92,27 @@ class SPATIALGDK_API SpatialRPCService
 	void OnEndpointAuthorityGained(Worker_EntityId EntityId, Worker_ComponentId ComponentId);
 	void OnEndpointAuthorityLost(Worker_EntityId EntityId, Worker_ComponentId ComponentId);
 
+	void SetRPCFailureOutcomesToQueue(const EPushRPCResult FailureOutcomes);
+
 private:
 	// For now, we should drop overflowed RPCs when entity crosses the boundary.
 	// When locking works as intended, we should re-evaluate how this will work (drop after some time?).
-	void ClearOverflowedRPCs(Worker_EntityId EntityId);
+	void ClearQueuedRPCs(Worker_EntityId EntityId);
 
 	EPushRPCResult PushRPCInternal(Worker_EntityId EntityId, ERPCType Type, RPCPayload&& Payload);
 
 	void ExtractRPCsForType(Worker_EntityId EntityId, ERPCType Type);
 
-	void AddOverflowedRPC(EntityRPCType EntityType, RPCPayload&& Payload);
+	void AddQueuedRPC(EntityRPCType EntityType, RPCPayload&& Payload);
 
 	uint64 GetAckFromView(Worker_EntityId EntityId, ERPCType Type);
 	const RPCRingBuffer& GetBufferFromView(Worker_EntityId EntityId, ERPCType Type);
 
 	Schema_ComponentUpdate* GetOrCreateComponentUpdate(EntityComponentId EntityComponentIdPair);
 	Schema_ComponentData* GetOrCreateComponentData(EntityComponentId EntityComponentIdPair);
 
+	bool ShouldQueueRPC(const EPushRPCResult Result, const ERPCType Type) const;
```

Comment:
This is cruft, as this function now lives in the PushRPCResultUtils namespace - will remove.